### DISCUSSION TITLE
Looking for evidence of new changes to metadata titles

### DISCUSSION THREAD

**Discussion by MarcelWilson** [python-poetry/poetry#8417] [Q&A] [answered]
I noticed there was a change in behavior in how a local package metadata that occurred in 1.5.1 and was subsequently fixed in 1.6.x but I'm not finding a issue ticket or the PR where the change occurred (assuming it was a code change in poetry at all)

I am one of the maintainers of [screenpy](https://github.com/ScreenPyHQ) and I recently noticed a newly [failing test](https://github.com/ScreenPyHQ/screenpy_selenium/actions/runs/5966029789/job/16184798124#step:5:46) in one of the plugins [screenpy_selenium](https://github.com/ScreenPyHQ/screenpy_selenium).  

```
      def test_metadata() -> None:
  >       assert __version__.__title__ == "screenpy-selenium"
  E       AssertionError: assert 'screenpy_selenium' == 'screenpy-selenium'
  E         - screenpy-selenium
  E         ?         ^
  E         + screenpy_selenium
  E         ?         ^
```
It would appear that the metadata created by the build no longer converts underscore to dash the name of a packge in the pyproject.toml.   From what I can tell it simply means we need to update our test, but I'd like to confirm that assertion.

I found this [lengthy discussion](https://discuss.python.org/t/change-in-pypi-upload-behavior-intentional-accidental-pebkac/27707/71) talking about how the community should proceed.  From what I can see the change in behavior in poetry 1.6.1 is consistent with their conversation.  This seems to further confirm my above hunch, but it's not definitive.   What I would like to be able to do is point to the specific change (issue, PR, commit, etc) but I'm not finding it.   Anyone able to point me in the right direction?

**Comment by radoering** [comment, marked as answer]
maybe python-poetry/poetry-core#620
